spring-cloud-gcp-logging/src/test/java/org/springframework/cloud/gcp/logging/it/StackdriverLoggingIntegrationTests.java

```diff
@@ -96,37 +97,32 @@ public void test() throws IOException {
 				"/", HttpMethod.GET, new HttpEntity<>(headers), String.class);
 		assertThat(responseEntity.getStatusCode().is2xxSuccessful()).isTrue();
 
-		CredentialsProvider credentialsProvider = this.credentialsProvider;
 		Logging logClient = LoggingOptions.newBuilder()
 				.setCredentials(credentialsProvider.getCredentials())
 				.build().getService();
 
-		await().atMost(60, TimeUnit.SECONDS).untilAsserted(() -> {
-			Page<LogEntry> page = logClient.listLogEntries(
-					Logging.EntryListOption.filter("textPayload:\"#$%^&" + NOW + "\" AND"
-							+ " logName=\"projects/" + this.projectIdProvider.getProjectId()
-							+ "/logs/spring.log\""));
-
-			assertThat(page.getValues()).hasSize(1);
-
-			LogEntry entry = page.getValues().iterator().next();
-			assertThat(entry.getTrace()).matches(
-					"projects/" + this.projectIdProvider.getProjectId() + "/traces/([a-z0-9]){32}");
-			assertThat(entry.getSpanId()).matches("([a-z0-9]){16}");
-		});
+		await().atMost(120, TimeUnit.SECONDS)
+				.pollInterval(Duration.FIVE_SECONDS)
+				.untilAsserted(() -> {
+					Page<LogEntry> page = logClient.listLogEntries(
+							Logging.EntryListOption.filter("textPayload:\"#$%^&" + NOW + "\" AND"
+									+ " logName=\"projects/" + this.projectIdProvider.getProjectId()
+									+ "/logs/spring.log\""));
+
+					assertThat(page.getValues()).hasSize(1);
+
+					LogEntry entry = page.getValues().iterator().next();
+					assertThat(entry.getTrace()).matches(
+							"projects/" + this.projectIdProvider.getProjectId() + "/traces/([a-z0-9]){32}");
+					assertThat(entry.getSpanId()).matches("([a-z0-9]){16}");
+				});
 	}
 
 	/**
 	 * web-app used for integration tests.
 	 */
 	@RestController
-	@SpringBootApplication(exclude = {
-			GcpCloudSqlAutoConfiguration.class,
-			GcpStorageAutoConfiguration.class,
-			DataSourceAutoConfiguration.class,
-			SecurityAutoConfiguration.class,
-			IapAuthenticationAutoConfiguration.class
-	})
+	@SpringBootApplication
```

Comment:
Since you're no longer using autoconfiguration, you'll probably need to manually configure the `TraceIdLoggingWebMvcInterceptor` bean. and the `TraceIdExtractor`.